demdata_cms/src/taccsite_cms/settings_custom.py

```diff
@@ -29,48 +20,10 @@
 # TACC: BRANDING
 ########################
 
-# LOOK INTO THIS SOLUTION.
-# from taccsite_cms.settings import TACC_BRANDING, UTEXAS_BRANDING, NSF_BRANDING
-
-TACC_BRANDING = [
-    "tacc",
-    "site_cms/img/org_logos/tacc-white.png",
-    "branding-tacc",
-    "https://www.tacc.utexas.edu/",
-    "_blank",
-    "TACC Logo",
-    "anonymous",
-    "True"
-]
-
-UTEXAS_BRANDING = [
-    "utexas",
-    "site_cms/img/org_logos/utaustin-white.png",
-    "branding-utaustin",
-    "https://www.utexas.edu/",
-    "_blank",
-    "University of Texas at Austin Logo",
-    "anonymous",
-    "True"
-]
-
-# NSF_BRANDING = [
-#     "nsf",
-#     "site_cms/img/org_logos/nsf-white.png",
-#     "branding-nsf",
-#     "https://www.nsf.gov/",
-#     "_blank",
-#     "NSF Logo",
-#     "anonymous",
-#     "True"
-# ]
-
-# BRANDING = [ TACC_BRANDING, UTEXAS_BRANDING, NSF_BRANDING ]
-BRANDING = [ TACC_BRANDING, UTEXAS_BRANDING ] # this matches prod 2022
-# BRANDING = [] # prod 2022 hides bar via snippet CSS but should also do this
+BRANDING = False
```

Comment:
**What?!**

1. The active setting is `BRANDING = [ TACC_BRANDING, UTEXAS_BRANDING ]`... which is also the default. So delete it and all it's content/variables.
2. The BRANDING is not desired (hence CSS hides it on prod site), so just set `BRANDING = False` to [prevent template from showing branding bar](https://github.com/TACC/Core-CMS/blob/v4.8.0/taccsite_cms/templates/header_branding.html#L4).